### DISCUSSION TITLE
Link labels

### DISCUSSION THREAD

**Discussion by joshmouch** [Blazor-Diagrams/Blazor.Diagrams#262] [General]
Is there a way to add labels to the links between nodes... even if it's something custom I have to add?  I think that's the only thing preventing me from trying this library.
Most of the other diagraming libraries I've tried support this such as Cytoscape, Mermaid, etc.

**Comment by zHaytam** [comment]
Hello,
Isn't that what [link labels](https://blazor-diagrams.zhaytam.com/links/labels) are for? If not, can you show me exactly what you need?

**Reply by joshmouch** [reply]
Yes, that will work!  Sorry, I completely missed that.
